docs/diff/rules.md

```diff
@@ -0,0 +1,190 @@
+# Rules for Diff Tool
+
+## Rules to categorize differences
+
+In rules, a difference can be categorized as Breaking or Non-Breaking.
+
+If no rules are provided a default set of rules will be applied on the differences.
+
+|Rule	|Category	|Description	|
```

Comment:
Are these default set of rules? If yes, maybe we should explicitly mention it.

Reply:
Fixed!